SFTP/node.listFiles.js

```diff
@@ -0,0 +1,19 @@
+let logger = global.logger;
+
+module.exports = async (connectorData, inputData) => {
+	logger.trace("SFTP List Files Node: Invoked!");
```

Comment:
We need an info level log that says which directory we are going to list

Reply:
Done!